frontend/src/pages/contracts/index.vue

```diff
@@ -6,9 +6,15 @@
 				<TableRow>
 					<TableTh>Address</TableTh>
 					<TableTh>Age</TableTh>
-					<TableTh align="right">Creator</TableTh>
+					<TableTh align="right">
+						Creator
+						<InfoTooltip text="Account address of contract creator"/>
+					</TableTh>
 					<TableTh align="right">Initial Transaction</TableTh>
-					<TableTh align="right">Module</TableTh>
+					<TableTh align="right">
+						Module
+						<InfoTooltip text="Container which holds execution code for one-or-more contracts. The below references holds the current execution code of the contract."/>
```

Comment:
This string repeats in a few places. It'd be better to define a constant for it.